api/v1/controllers/subjects.js

```diff
@@ -24,7 +24,7 @@ const doGetHierarchy = require('../helpers/verbs/doGetHierarchy');
 const doPatch = require('../helpers/verbs/doPatch');
 const doPost = require('../helpers/verbs/doPost');
 const doPut = require('../helpers/verbs/doPut');
-const u = require('../helpers/verbs/utils');
```

Comment:
Added my renaming stuffs, but next commit I have changed to be like the original master convention (using the variable as 'u');
Just added my stuff on next commit, leaving the code as much close to the original master branch